python/python/glide/routes.py

```diff
@@ -48,6 +48,24 @@ def __init__(self, slot_type: SlotType, slot_id: int) -> None:
         self.slot_id = slot_id
 
 
+class ByAddressRoute(Route):
+    def __init__(self, host: str, port: Optional[int] = None) -> None:
+        """Routes a request to a node by its DNS address
```

Comment:
I don't think DNS is right, as it can be also IP.
maybe change to 
```
Routes a request to a node based on its host address and IP

Reply:
when/how can it be IP? I'm not sure that it will work. Also, why would the user have the IP of the machine?

Reply:
how will we support when nodes dont have the dns but only the IP ?

Reply:
I think it depends on how user configured the cluster. If nodes know each other by IP, so response of `cluster slots` would contain IPs only and glide core would know IPs only.
Probably, we need to refer there to cluster configuration, like 
```suggestion
        """Routes a request to a node by its address given on cluster configuration
```

Reply:
> when/how can it be IP? I'm not sure that it will work. Also, why would the user have the IP of the machine?
If the cluster is configured with TLS disabled, the nodes will be shown in cluster slots/nodes with their IPs, rather than their DNS. Yuri is right that it depends on server.